packages/jss-preset-default/src/index.js

```diff
@@ -0,0 +1,40 @@
+// @flow
+
+import template from 'jss-template'
+import global from 'jss-global'
+import extend from 'jss-extend'
+import nested from 'jss-nested'
+import compose from 'jss-compose'
+import camelCase from 'jss-camel-case'
+import defaultUnit from 'jss-default-unit'
+import expand from 'jss-expand'
+import vendorPrefixer from 'jss-vendor-prefixer'
+import propsSort from 'jss-props-sort'
+
+type Options = {
+  template?: {},
```

Comment:
in theory once all other plugins are typed, you could import each type from there and use it here